knowledge_graph/make_graph.py

```diff
@@ -255,73 +167,6 @@ def get_test_ontology(G, valid_test_ont, not_test_ont):
         remove_non_test_nodes(G, node_b, valid_test_ont, not_test_ont)
 
 
-def get_valid_test_ont():
```

Comment:
We need some utils function to pull out things that are test ontology content for the designer to have a simplified JSON file she can read through to know what content exists in the knowledge graph that could be used in designs. So if deleting this function, then we need to write another function. Also, this gets used in unit testing I think for CI/CD (another reason it would need to be replaced with better code or kept/moved to utils).

Reply:
I didn't delete this, I moved it to utils. Additionally, it gets imported into makge_graph.py at line 10, so it's still importable from make_graph as though it was defined here. But while we're discussing this, I was planning on refactoring this a bit more. We don't need a function to return a collection like this, we could just define a variable `TEST_ONTOLOGY = {...}` and import that. But I actually don't think that's the right way to do it either: a flat list of values like this belongs in a text file. My plan is to move these values to a text file and read the file into `get_valid_test_ont()`. This way any downstream functions (e.g. CI/CD) that call get_valid_test_ont will continue to function as expected, but also we'll have clearer change management of what values we consider to be test ontology (e.g. we can modify this list without making it look like we changed any utils functions). Additionally, it will make those values available more generally, so they aren't locked into the python component of the application.